### PR TITLE
make sure multiwsidatasets are validated

### PR DESCRIPTION
**Pull request by roman807**
closes https://github.com/kaiko-ai/eva/issues/499

### CODE REVIEW

File: src/eva/core/data/datasets/base.py [hunk 1]
```diff
@@ -26,6 +26,10 @@ def setup(self) -> None:
         """
         self.configure()
         self.validate()
+        self._setup()
+
+    def _setup(self) -> None:
```
Comment (ioangatop) [inline src/eva/core/data/datasets/base.py:31]: why we have this method? for setups we should use the `self.configure()` method, or is this a blocker?

Comment (roman807) [reply src/eva/core/data/datasets/base.py:31]: i see, @nkaenzig do you see anything that speaks against removing `_setup` and replacing it with `configure` in `MultiWsiDataset`?

Comment (nkaenzig) [reply src/eva/core/data/datasets/base.py:31]: I think using configure is fine, I had forgotten this existed :D

Comment (roman807) [reply src/eva/core/data/datasets/base.py]: move validate to configure

**Pull request closed without merging**